### PR TITLE
fix(id/doujindesu): fix chapter does not load

### LINKED ISSUE #14004: Doujindesu: Chapter images does not load
### Source information and language

Doujindesu 1.3.3/Indonesian

### Steps to reproduce

1) open the doujindesu extension 
2) select a comic 
3) open the chapter 
4) no image found

### Expected behavior

Please fix the doujindesu extension 
The image doesn't appear in the chapter

### Actual behavior

Image not found
Screenshot_2022-10-28-01-36-19-01_afd66b7916d19a60815f56273408b73e

### Tachiyomi version

0.13.2

### Android version

Android 9

### Other details

_No response_

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.13.6](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by Sofie99**
Closes #14004

Checklist:

- [x] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [ ] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio

### CODE REVIEW

**Comment by TheKingTermux** [comment]
I forget to mention it on #14004
The chapter name is missing like this image

There is no `"Chapter {number} {doujin_title}"` or `"Chapter {number}"` in there and for example when it is downloaded, it will generate a `"(invalid)"` folder on storage, there is only a *date* there, but that date *does not follow* the date of the *chapter that was created / uploaded* from DoujinDesu, but follows the date when we *first opened Doujin*
Has it been fixed?

**Comment by TheKingTermux** [comment]
> This issues has been solved too..
> 

Uumm.... Where the release date ?
There was an error in the upload / release date

**Comment by TheKingTermux** [comment]
> @TheKingTermux I've fixed date, popular and new.
> Everything should work fine now
> 
> 

That's what I mean, thanks for fixing it, sorry if I'm a bother, because I don't have any laptop / PC available that I can use to fix or testing this extension since December last year

**Comment by AhriVastayan1** [comment]
> > @TheKingTermux I've fixed date, popular and new.
> > Everything should work fine now
> > 
> 
> That's what I mean, thanks for fixing it, sorry if I'm a bother, because I don't have any laptop / PC available that I can use to fix or testing this extension since December last year, LGTM to approve, but idk when the moderator review it

How to solve that man, it say page not found

**Comment by TheKingTermux** [comment]
> How to solve that man, it say page not found

Just wait, updates will be available when this PR has been approved and merged by the moderator

**Comment by TheKingTermux** [comment]
> How to solve that man, it say page not found

Just wait, updates will be available when this PR has been approved and merged by the moderator

File: src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt [hunk 1]
```diff
@@ -261,8 +267,8 @@ class DoujinDesu : ParsedHttpSource() {
         }
     }
 
-    private fun getNumberFromString(epsStr: String): String {
-        return epsStr.filter { it.isDigit() }
+    private fun getNumberFromString(epsStr: String?): Float {
+        return epsStr?.filter { it.isDigit() }?.toFloatOrNull() ?: 1F
```
Comment (alessandrojean) [inline src/id/doujindesu/src/eu/kanade/tachiyomi/extension/id/doujindesu/DoujinDesu.kt:271]: Should instead default to `-1f`.

```suggestion
        return epsStr?.filter { it.isDigit() }?.toFloatOrNull() ?: -1f
```

**Review by alessandrojean — CHANGES_REQUESTED** [review]

**Pull request merged**